backend/tests/v5/e2e/routes/teamspaces/projects/containers/containers.test.js

```diff
@@ -183,6 +183,39 @@ const testGetContainerStats = () => {
 	});
 };
 
+const testAppendFavourites = () => {
+	const route = `/v5/teamspaces/${teamspace}/projects/${project.id}/containers/favourites`;
+	describe('Append Favourite Containers', () => {
+		test('should fail without a valid session', async () => {
+			const res = await agent.get(route).expect(templates.notLoggedIn.status);
```

Comment:
you're not testing your route?

this is a `HTTP get`, not a `HTTP patch` operation